draft-ietf-quic-recovery.md

```diff
@@ -309,7 +309,8 @@ parameter, see Section 18.1 of {{QUIC-TRANSPORT}}.  max_ack_delay implies an
 explicit contract: an endpoint promises to never delay acknowledgments of an
 ack-eliciting packet by more than the indicated value. If it does, any excess
 accrues to the RTT estimate and could result in spurious retransmissions from
-the peer.
+the peer.  Before receiving the transport parameters, endpoints use a value of 0
```

Comment:
This discusses transport param defaults, not Initial and Handshake packets.  Did you intend to keep this?